orchagent/portsorch.cpp

```diff
@@ -1250,75 +1494,72 @@ bool PortsOrch::removeVlan(Port vlan)
         return false;
     }
 
-    sai_status_t status = sai_vlan_api->remove_vlan(vlan.m_vlan_oid);
+    sai_status_t status = sai_vlan_api->remove_vlan(vlan.m_vlan_info.vlan_oid);
     if (status != SAI_STATUS_SUCCESS)
     {
-        SWSS_LOG_ERROR("Failed to remove VLAN %s vid:%hu", vlan.m_alias.c_str(), vlan.m_vlan_id);
+        SWSS_LOG_ERROR("Failed to remove VLAN %s vid:%hu",
+                vlan.m_alias.c_str(), vlan.m_vlan_info.vlan_id);
         return false;
     }
 
-    SWSS_LOG_NOTICE("Remove VLAN %s vid:%hu", vlan.m_alias.c_str(), vlan.m_vlan_id);
+    SWSS_LOG_NOTICE("Remove VLAN %s vid:%hu", vlan.m_alias.c_str(),
+		vlan.m_vlan_info.vlan_id);
 
     m_portList.erase(vlan.m_alias);
 
     return true;
 }
 
-bool PortsOrch::addVlanMember(Port vlan, Port port, string& tagging_mode)
+bool PortsOrch::addVlanMember(Port &vlan, Port &port, string& tagging_mode)
```

Comment:
align the space, e.g. `string &tagging_mode`

Reply:
ok